### PR TITLE
CO-2708: updating the traduction file with poeditor. there is still s…

### PR DESCRIPTION
**Pull request by samybucher**
…ome files missing in the french dossier, german didnt come in the commit, I'll check this tomorrow

### CODE REVIEW

**Comment by samybucher** [comment]
with german file this time!

File: child_compassion/i18n/de.po [hunk 1]
```diff
@@ -977,6 +991,14 @@ msgid ""
 "entrepreneurship, and handicrafts. Caregivers receive training on child care "
 "from the church as well as bi-annual medical check-ups and church activities."
 msgstr ""
+"ie Kinder können jedes Jahr ausserhalb des Lehrplans ein Lager besuchen. Sie "
```
Comment (ecino) [inline child_compassion/i18n/de.po]:
```suggestion
"Die Kinder können jedes Jahr ausserhalb des Lehrplans ein Lager besuchen. Sie "
```

File: child_compassion/i18n/de.po [hunk 2]
```diff
@@ -996,6 +1024,11 @@ msgid ""
 "some critical situations where children from extremely poor families are "
 "given more food to take home."
 msgstr ""
+"ie Kinder in Togo erhalten jeden Tag, an dem sie ins Zentrum kommen, ein "
```
Comment (ecino) [inline child_compassion/i18n/de.po]:
```suggestion
"Die Kinder in Togo erhalten jeden Tag, an dem sie ins Zentrum kommen, ein "
```

File: child_compassion/i18n/de.po [hunk 3]
```diff
@@ -2008,6 +2047,11 @@ msgid ""
 "such as silk screening, jewellery-making, baking and buffet preparation. "
 "Parents are offer monthly parenting classes."
 msgstr ""
+"eden Monat wird eine Campingzeit angeboten, an der die Kinder teilnehmen "
```
Comment (ecino) [inline child_compassion/i18n/de.po]:
```suggestion
"Eden Monat wird eine Campingzeit angeboten, an der die Kinder teilnehmen "
```
Beginning of sentence, should it be a capital letter?

File: child_compassion/i18n/de.po [hunk 4]
```diff
@@ -3662,6 +3754,8 @@ msgid ""
 "Kids and young people are given the chance to have fun, develop "
 "relationships and play in a safe space."
 msgstr ""
+"ie Kinder und Jugendlichen haben Spass, knüpfen Freundschaften und können in "
```
Comment (ecino) [inline child_compassion/i18n/de.po]:
```suggestion
"Die Kinder und Jugendlichen haben Spass, knüpfen Freundschaften und können in "
```

File: child_compassion/i18n/de.po [hunk 5]
```diff
@@ -3670,6 +3764,9 @@ msgid ""
 "relationships and play in a safe space. A typical snack may be a cup of "
 "porridge, a slice of bread, fruit or an egg."
 msgstr ""
+"ie Kinder und Jugendlichen haben Spass, knüpfen Freundschaften und können in "
```
Comment (ecino) [inline child_compassion/i18n/de.po]:
```suggestion
"Die Kinder und Jugendlichen haben Spass, knüpfen Freundschaften und können in "
```

File: child_compassion/i18n/de.po [hunk 6]
```diff
@@ -5232,6 +5344,9 @@ msgid ""
 "Project tutors share valuable life skills e.g. how to resolve conflict and "
 "the importance of developing healthy self-esteem."
 msgstr ""
+"ie Mitarbeitenden des Kinderzentrums vermitteln wertvolle Lebensweisheiten, "
```
Comment (ecino) [inline child_compassion/i18n/de.po]:
```suggestion
"Die Mitarbeitenden des Kinderzentrums vermitteln wertvolle Lebensweisheiten, "
```

**Review by ecino — CHANGES_REQUESTED** [review]
Thanks! Some letters are missing.

**Comment by ecino** [comment]
Thank you. I'm waiting for the missing translations and we will merge afterwards. You forgot as well to fix the missing letters in German, can you look at it?

**Comment by ecino** [comment]
Missing translation for Honduras:
```
Les repas servis aux enfants sont en principe constitués de viande, de légumes, de céréales, de fruits et de jus de fruits. Les enfants souffrant de malnutrition bénéficient de compléments alimentaires, selon les indications données par le personnel médical.
```

**Comment by ecino** [comment]
For the other translation, Christian told me the text is not useful. So you can just put a space as the translation so that it won't display anything.

**Review by ecino — APPROVED** [review]
:+1:

**Pull request merged**